src/dsp_tools/commands/rosetta.py

```diff
@@ -48,8 +48,7 @@ def _clone_repo(
     """
     print(f"Clone into {rosetta_folder}...")
     completed_process = subprocess.run(
-        "git clone https://github.com/dasch-swiss/082E-rosetta-scripts.git",
-        shell=True,
+        "git clone https://github.com/dasch-swiss/082E-rosetta-scripts.git".split(),
         cwd=enclosing_folder,
         check=False,
     )
```

Comment:
IMO this would look nicer as two one-liners:
```python
    cmd =  "git clone https://github.com/dasch-swiss/082E-rosetta-scripts.git".split()
    completed_process = subprocess.run(cmd, cwd=enclosing_folder, check=False,)
```

Reply:
by the way, this could be done in a less low-level (but maybe not less complicated) way by using a library like `GitPython`